packages/web-react/src/types/box.ts

```diff
@@ -0,0 +1,47 @@
+import { ElementType, JSXElementConstructor } from 'react';
+import {
+  BackgroundColorsDictionaryType,
+  BorderColorsDictionaryType,
+  BreakpointToken,
+  ChildrenProps,
+  SpaceToken,
+  SpiritPolymorphicElementPropsWithRef,
+  StyleProps,
+} from './shared';
+
+export interface BoxBaseProps extends ChildrenProps, StyleProps {
+  /** The background color of the box. */
+  backgroundColor?: BackgroundColorsDictionaryType;
+  /** The border color of the box. */
+  borderColor?: BorderColorsDictionaryType;
+  /** The border radius of the box. */
+  borderRadius?: string;
```

Comment:
can we read this type from tokens? so we don't allow string, but allow only the radii we have?

Reply:
same for width? But there I think it might be a problem, because we don't have a map of widths.